### DISCUSSION TITLE
XTC Next Release

### DISCUSSION THREAD

**Discussion by GUOLDEV** [xamarin/XamarinCommunityToolkit#1657] [Q&A] [answered]
Hi All,

Is there a plan for the next XTC release?

Best regards,

GUOLDEV

**Comment by djl-85** [comment]
Hi all,
I am wondering the same. Because this nuget is not updated to the latest xamarin forms version, we cannot update the xamarin forms nuget, and we are missing important fixes, such as memory leaks in android views.

Thank you and best regards.

**Reply by IeuanWalker** [reply]
@djl-85 you should be able to update XF. NuGet just specifies the minimum version of XF required

**Reply by djl-85** [reply]
Thank you for your answer @IeuanWalker , but unfortunately in the last version of the Xamarin.Toolkit, that is not as you said.
![xamarin toolkit](https://user-images.githubusercontent.com/58255204/138403248-a85130c9-88a4-4f82-aaba-6d190f8cda66.png)

Thank you.

**Comment by kocevilija** [comment]
Hello,

I have the same issue regarding `Xamarin.Forms` dependency. I am using `Xamarin.CommunityToolkit` for multiple purposes in my projects, however other packages require newer version than `5.0.0.2012` of `Xamarin.Forms`. So I need to choose between packages and I need and love `Xamarin.CommunityToolkit` :).

When can we expect update regarding this issue?

**Reply by pictos** [reply]
Hello there, we will have a stable release soon.

**Comment by pictos** [comment, marked as answer]
We are working on it. Will be release soon.(:

**Reply by GUOLDEV** [reply]
That's great 🎉 thanks @pictos for the update 👍
